app/lib/service/entrypoint/frontend.dart

```diff
@@ -52,6 +53,7 @@ Future _main() async {
   await announcementBackend.start();
   await topPackages.start();
   await youtubeBackend.start();
+  await downloadCountsBackend.start();
```

Comment:
@isoos is this this the right place to start it?

unrelated: Should we investigate if these could be started concurrently?

Reply:
I think it is a good place to start it, unless we only want to start it in the `default` and not in the others.